src/main/resources/application.yaml

```diff
@@ -85,10 +85,16 @@ rest:
 "%dev":
   damap:
     origins: http://localhost:4200
+    auth:
+      backend:
+        url: http://localhost:8087/auth/realms/damap # https://your.authentication.server
     datasource:
-      url: jdbc:postgresql://localhost:5432/damap
+      url: jdbc:postgresql://localhost:8088/damap
```

Comment:
With all these changes, it also makes sense to remove the comment on the port and map the port 8088 open by default for the `damap-db` service in the `docker-compose` file.
https://github.com/tuwien-csd/damap-backend/blob/c9e43f8cd4c742652fa7162148bea986af119aa1/docker/docker-compose.yaml#L21-L24

Reply:
Yes that is a good idea. Out of curiosity, what was the reason for leaving it commented out?